src/main/java/net/snowflake/client/jdbc/RestRequest.java

```diff
@@ -261,17 +282,20 @@ public static CloseableHttpResponse execute(
       } catch (Exception ex) {
 
         savedEx = ex;
-        // if the request took more than 5 min (socket timeout) log an error
+        // if the request took more than socket timeout log an error
         if ((System.currentTimeMillis() - startTimePerRequest)
             > HttpUtil.getSocketTimeout().toMillis()) {
           logger.warn(
-              "HTTP request took longer than socket timeout {} s: {} s",
-              HttpUtil.getSocketTimeout().toMillis() / 1000, (System.currentTimeMillis() - startTimePerRequest) / 1000);
+              "{}HTTP request took longer than socket timeout {} s: {} s",
+              requestIdStr,
+              HttpUtil.getSocketTimeout().toMillis() / 1000,
```

Comment:
do we want to have seconds or milliseconds here?

Reply:
After thinking I think that ms would be better. Changed.